packages/devtools_app/lib/src/shared/config_specific/server/_server_stub.dart

```diff
@@ -61,10 +57,6 @@ Future<String> setLastShownReleaseNotesVersion(String version) async {
   throw Exception(unsupportedMessage);
 }
 
-Future<void> resetDevToolsFile() async {
```

Comment:
can we revert this change and the same for the _server_web.dart file below? This is the DevTools API to access the DevTools server, so even if it is not used right now, leaving this in the interface is still useful to expose the DevTools server endpoints. (kind of like the vm_service_wrapper.dart file - we don't use every method, but we are implementing a protocol so we leave them all there)

Reply:
Yes, please add a comment. Thanks.

Reply:
Done